src/graphql/data/fetchGraphQL.ts

```diff
@@ -2,17 +2,13 @@ import { Variables } from 'react-relay'
 import { GraphQLResponse, RequestParameters } from 'relay-runtime'
 
 const TOKEN_URL = process.env.REACT_APP_AWS_API_ENDPOINT
-const NFT_URL = process.env.REACT_APP_NFT_AWS_API_ENDPOINT ?? ''
+const NFT_URL = process.env.REACT_APP_NFT_AWS_API_ENDPOINT
+if (!TOKEN_URL) throw new Error('Token URL missing from environment')
```

Comment:
should this be moved to `fetchToken` for consistency?

Reply:
Actually I came in here to leave the same question, but the other way around! If the intent is to `throw on missing env var URLs.`, shouldn't we throw on a top-level to break early?

If I am not mistaken, this code is bundled and then, run client side, so the only time where this can happen is while development and/or bundling.

Reply:
One additional suggestion, we could move all environmental variables into a single `env` file. Not only would it be easier to handle missing variables, but also to provide proper Typescript support

Reply:
the NFT queries are hidden behind the feature flag so we wouldn't want missing that endpoint to break the default site